grafana-plugin/src/pages/settings/tabs/ChatOps/tabs/SlackSettings/SlackSettings.tsx

```diff
@@ -189,16 +211,6 @@ class SlackSettings extends Component<SlackProps, SlackState> {
     );
   };
 
-  renderActionButtons = () => {
-    <WithPermissionControl userAction={UserActions.ChatOpsUpdateSettings}>
-      <WithConfirm title="Are you sure to delete this Slack Integration?">
-        <Button variant="destructive" size="sm" onClick={() => this.removeSlackIntegration()}>
-          Disconnect
-        </Button>
-      </WithConfirm>
-    </WithPermissionControl>;
-  };
-
```

Comment:
`renderActionButtons` was essentially a duplicate of what was changed + it was not consumed within this component